### PR TITLE
Add depth to perform shallow fetch.

### PR DESCRIPTION
**Pull request by headw01**
Just enough to support a shallow fetch for our needs.

### CODE REVIEW

File: org.eclipse.jgit/src/org/eclipse/jgit/api/FetchCommand.java [hunk 1]
```diff
@@ -450,6 +453,40 @@ public FetchCommand setDryRun(boolean dryRun) {
...
+	/**
+	 * Get the set depth value.
+	 * @return The set depth
+	 */
+	public int getDepth() {
+		return depth;
+	}
+
+	/**
+	 * Sets whether the fetch depth.
```
Comment (chrismuellercert) [inline org.eclipse.jgit/src/org/eclipse/jgit/api/FetchCommand.java]: Should this just be "Sets the fetch depth."?

**Review by chrismuellercert — APPROVED** [review]
Not super familiar with how this works but the changes look reasonable to me.

**Review by searlm — APPROVED** [review]
Looks reasonable enough to me.

File: org.eclipse.jgit/src/org/eclipse/jgit/api/FetchCommand.java (hunk 1 above)
Comment (headw01) [reply org.eclipse.jgit/src/org/eclipse/jgit/api/FetchCommand.java]: Yep, thanks.

**Pull request merged**